Makefile

```diff
@@ -66,7 +62,13 @@ test: clean ## Run tests and generate coverage report
 	${VENV_BIN}/coverage xml
 	${VENV_BIN}/diff-cover coverage.xml --html-report diff-cover.html
 
-testserver:  ## Run an isolated ephemeral instance of Blockstore for use by edx-platform tests
+easyserver: dev.up dev.provision  # Start and provision a Blockstore container and run the server until CTRL-C, then stop it
+	# Now run blockstore until the user hits CTRL-C:
+	docker-compose --project-name blockstore -f docker-compose.yml exec blockstore /blockstore/venv/bin/python /blockstore/app/manage.py runserver 0.0.0.0:18250
```

Comment:
Interesting, would be useful for the other devstack as well if could pass arbitrary commands to be run inside the container.